examples/platforms/posix/uart.c

```diff
@@ -188,7 +188,7 @@ void posixUartUpdateFdSet(fd_set *aReadFdSet, fd_set *aWriteFdSet, int *aMaxFd)
 
 void posixUartProcess(void)
 {
-    const int flags = POLLRDNORM | POLLERR | POLLNVAL | POLLHUP;
+    const int flags = POLLIN | POLLRDNORM | POLLERR | POLLNVAL | POLLHUP;
```

Comment:
A little curious about this change. It looks like an oversight on our part (Not sure why we weren't doing `POLLIN` to begin with). Were you running into problems without this change? It might be worthwhile to break this out into a separate pull request. (If you do, just remove `POLLRDNORM`, since `POLLIN` implies it)